tests/bpf2c_tests/expected/divide_by_zero_dll.c

```diff
@@ -154,12 +154,7 @@ divide_by_zero(void* context)
     r6 = IMMEDIATE(100000);
     // EBPF_OP_DIV64_REG pc=10 dst=r6 src=r1 offset=0 imm=0
 #line 32 "sample/divide_by_zero.c"
-    if (r1 == 0) {
-        division_by_zero(10);
-        return 0xffffffffffffffffui64;
-    }
-#line 32 "sample/divide_by_zero.c"
-    r6 /= r1;
+    r6 = r1 ? (r6 / r1) : 0;
```

Comment:
Should we add a sample program for mod also?

Reply:
That path is covered by the tests we pull from uBPF and then run through bpf2c.